wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
+| Hyper Potions | Clear all the beatmaps bundled in the [Hyper Potions Pack](https://osu.ppy.sh/beatmaps/packs/2037) beatmap pack. |
+| Kola Kid | Clear all the beatmaps bundled in the [Kola Kid Pack](https://osu.ppy.sh/beatmaps/packs/2044) beatmap pack. |
+| LeaF | Clear all the beatmaps bundled in the [LeaF Pack](https://osu.ppy.sh/beatmaps/packs/2039) beatmap pack. |
+| Panda Eyes | Clear all the beatmaps bundled in the [Panda Eyes Pack](https://osu.ppy.sh/beatmaps/packs/2043) beatmap pack. |
+| PUP | Clear all the beatmaps bundled in the [PUP Pack](https://osu.ppy.sh/beatmaps/packs/2048) beatmap pack. |
+| Ricky Montgomery | Clear all the beatmaps bundled in the [Ricky Montgomery Pack](https://osu.ppy.sh/beatmaps/packs/2046) beatmap pack. |
+| Rin | Clear all the beatmaps bundled in the [Rin Pack](https://osu.ppy.sh/beatmaps/packs/1759) beatmap pack. |
+| S3RL | Clear all the beatmaps bundled in the [S3RL Pack](https://osu.ppy.sh/beatmaps/packs/2045) beatmap pack. |
+| Sound Souler | Clear all the beatmaps bundled in the [Sound Souler Pack](https://osu.ppy.sh/beatmaps/packs/2038) beatmap pack. |
+| Teminite | Clear all the beatmaps bundled in the [Teminite Pack](https://osu.ppy.sh/beatmaps/packs/2042) beatmap pack. |
+| VINXIS | Clear all the beatmaps bundled in the [VINXIS Pack](https://osu.ppy.sh/beatmaps/packs/2041) beatmap pack. |
+| Mappers' Guild Pack I | Clear all the beatmaps bundled in the [Mappers' Guild Pack I](https://osu.ppy.sh/beatmaps/packs/1365) beatmap pack. |
+| Mappers' Guild Pack II | Clear all the beatmaps bundled in the [Mappers' Guild Pack II](https://osu.ppy.sh/beatmaps/packs/1450) beatmap pack. |
+| Mappers' Guild Pack III | Clear all the beatmaps bundled in the [Mappers' Guild Pack III](https://osu.ppy.sh/beatmaps/packs/1689) beatmap pack. |
+| Mappers' Guild Pack IV | Clear all the beatmaps bundled in the [Mappers' Guild Pack IV](https://osu.ppy.sh/beatmaps/packs/1757) beatmap pack. |
+| Mappers' Guild Pack V | Clear all the beatmaps bundled in the [Mappers' Guild Pack V](https://osu.ppy.sh/beatmaps/packs/2032) beatmap pack. |
+| Mappers' Guild Pack VI | Clear all the beatmaps bundled in the [Mappers' Guild Pack VI](https://osu.ppy.sh/beatmaps/packs/2033) beatmap pack. |
 
-- Mods can be used except for mods that can cause unranked play (e.g. [Auto Pilot](/wiki/AP)).
-- The beatmap can be deleted safely once the **`CLEARED`** status was noted in the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=theme).
+It is worth noting however that the contents of all the Video Game Packs, Anime Packs, Internet! Packs, and Rhythm Game Packs have been dramatically altered over time since [their first introduction in 2008](https://osu.ppy.sh/community/forums/topics/1853). For the list of all the beatmaps that was being listed in the aforementioned beatmap packs in their legacy/original state, see [Legacy Beatmap Packs](/wiki/Medals/Legacy_Beatmap_Packs).
```

Comment:
```suggestion
The contents of all the Video Game Packs, Anime Packs, Internet! Packs, and Rhythm Game Packs have been dramatically altered over time since [their first introduction in 2008](https://osu.ppy.sh/community/forums/topics/1853). For a list of all the beatmaps that are listed in the aforementioned beatmap packs in their legacy/original state, see [Legacy Beatmap Packs](/wiki/Medals/Legacy_Beatmap_Packs).
```
The first sentence should probably be formatted as a note and the second sentence should probably be formatted as a for see hatnote